lib/Elastica/Multi/MultiBuilder.php

```diff
@@ -38,16 +38,15 @@ private function buildResultSet(Response $childResponse, BaseSearch $search)
      */
     private function buildResultSets(Response $response, $searches)
     {
-        $resultSets = array();
-
         $data = $response->getData();
-        if (!isset($data['responses']) || !is_array($data['responses'])) {
-            return $resultSets;
+        if (empty($data['responses']) || !is_array($data['responses'])) {
```

Comment:
If see all method - and use `!isset($data['responses'])` instead `empty($data['responses']))` - so we have `foreach ([] as $responseData) {}` so use `empty($data['responses']))` - remove this foreach.
The result of method - same, but in my - without `reset` and `empty foreach`